modular_skyrat/modules/ashwalkers/code/effects/ash_rituals.dm

```diff
@@ -229,27 +236,106 @@
 			continue
 		valid_humans += human_share
 		divide_damage++
-	//how much damage each person will take
+
 	var/singular_damage = total_damage / divide_damage
-	//do the healing and damage taking
 	for(var/mob/living/carbon/human/human_target in valid_humans)
 		human_target.adjustBruteLoss(singular_damage)
 	human_victim.heal_overall_damage(human_victim.getBruteLoss(), human_victim.getFireLoss())
+
+/datum/ash_ritual/banish_kin
+	name = "Banish Kin"
+	desc = "Some kin are not fit for the tribe, this can solve that issue through democracy."
+	required_components = list()
+	consumed_components = list()
+
+/datum/ash_ritual/banish_kin/ritual_success(obj/effect/ash_rune/success_rune)
+	. = ..()
+	var/turf/src_turf = get_turf(success_rune)
+
+	var/mob/living/carbon/human/find_banished = locate() in src_turf
+	if(!find_banished)
+		return
+
+	if(!find_banished.mind.has_antag_datum(/datum/antagonist/ashwalker)) //must be an ashwalker
+		return
+
+	var/list/asked_voters = list()
+
+	for(var/mob/living/carbon/human/poll_human in range(2, src_turf))
+		if(poll_human.stat != CONSCIOUS) //must be conscious
+			continue
+
+		if(!poll_human.mind.has_antag_datum(/datum/antagonist/ashwalker)) //must be an ashwalker
+			continue
+
+		asked_voters += poll_human
+
+	var/list/yes_voters = poll_candidates("Do you wish to banish [find_banished]?", poll_time = 10 SECONDS, group = asked_voters)
+
+	if(length(yes_voters) < length(asked_voters))
+		find_banished.balloon_alert_to_viewers("banishment failed!")
+		return
+
+	var/turf/teleport_turf = locate(rand(1,255), rand(1,255), success_rune.z)
+
+	var/anti_endless = 0
+	while(!istype(teleport_turf, /turf/open/misc/asteroid) && anti_endless < 100)
+		teleport_turf = locate(rand(1,255), rand(1,255), success_rune.z)
+		anti_endless++
+
+	new /obj/effect/particle_effect/sparks(teleport_turf)
+	find_banished.forceMove(teleport_turf)
+
+/datum/ash_ritual/revive_animal
+	name = "Revive Animal"
+	desc = "Revives a simple animal that will then become friendly."
+	required_components = list(
+		"north" = /obj/item/organ/internal/regenerative_core,
+		"south" = /obj/item/organ/internal/regenerative_core,
+		"east" = /obj/item/stack/sheet/bone,
+		"west" = /obj/item/stack/sheet/sinew,
+	)
+	consumed_components = list(
+		/obj/item/organ/internal/regenerative_core,
+		/obj/item/stack/sheet/bone,
+		/obj/item/stack/sheet/sinew,
+	)
+
+/datum/ash_ritual/revive_animal/ritual_success(obj/effect/ash_rune/success_rune)
+	. = ..()
+	var/turf/src_turf = get_turf(success_rune)
+
+	var/mob/living/simple_animal/find_animal = locate() in src_turf
+
+	if(!find_animal)
+		return
+
+	if(find_animal.stat != DEAD)
+		return
+
+	if(find_animal.sentience_type != SENTIENCE_ORGANIC)
+		return
+
+	find_animal.faction = list("neutral")
+
+	if(ishostile(find_animal))
+		var/mob/living/simple_animal/hostile/hostile_animal = find_animal
+		hostile_animal.attack_same = FALSE
+
+	find_animal.revive(full_heal = TRUE, admin_revive = TRUE)
```

Comment:
Why does it need admin_revive = TRUE?

Reply:
Same way that lazarus injectors have it set to be admin_revive = TRUE.